css/main.css

```diff
@@ -0,0 +1,474 @@
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+/* CONTACT FORM */
+.contact {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0 16px;
+  padding-top: 90px;
+  gap: 20px;
+}
+
+.contact-heading {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 50px;
+}
+
+.contact-form {
+  margin-bottom: 150px;
+}
+
+.contact-form,
+.contact-group {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 14px;
+}
+
+.contact-group {
+  border: 0;
+}
+
+label {
+  visibility: hidden;
+}
+
+.contact-group input,
+.contact-group textarea {
+  width: 100%;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 36px;
+}
+
+.contact-form input {
+  border-bottom: 1px solid #dbd8d7;
+  padding: 15px 16px;
+}
+
+.contact-form textarea {
+  background-color: #fbf8f7;
+  padding: 12px 12px;
+}
+
+.contact-group input:focus,
+.contact-group textarea:focus {
+  outline: none;
+}
+
+.contact-form textarea::placeholder {
+  color: #3c3a39;
+}
+
+.btn-form {
+  border: none;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  cursor: pointer;
+  transition: all 0.7s;
+  position: relative;
+}
+
+.btn-form:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.footer-social-icons {
+  width: 100%;
+  border-top: 1px solid #dfe1e6;
+  padding-top: 24px;
+  padding-bottom: 34px;
+}
+
+.social-icons-list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2.4rem;
+}
+
+@media screen and (min-width: 1200px) {
+  * {
+    overflow-x: initial;
+  }
+
+  .contact-box {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 45px;
+    padding-top: 60px;
+  }
```

Comment:
- [ ] Great job styling your .contact-box class✔️. However, please kindly make sure that you center your form in order to stick as much as possible to the Figma design as requested in the requirement below 👇🏽.
  >You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/233164326-239e6927-68d0-4800-b425-a7515bc0a60e.png)

This is your design 👇🏽 (almost there😅)

![design](https://user-images.githubusercontent.com/116701485/233165014-c446e731-fe75-4986-8acf-ef6f6c8834d6.png)